scripts/tests/scripts_regression_tests.py

```diff
@@ -3537,6 +3553,10 @@ def _main_func(description):
                         help="Do not submit jobs to batch system, run locally."
                         " If false, will default to machine setting.")
 
+    parser.add_argument("--no-run", action="store_true",
+                        help="Do not run any fortran jobs."
+                        " used for github actions")
```

Comment:
The name of this argument (and the related variables in the code) makes it sound more general than it is (based on the help description). Would something like `--no-compiled-tests` or `--no-fortran-tests` or `--python-only` (or something similar) be more appropriate? Also, I'm unclear how this relates to the `--fast` argument; can you clarify that? Also, I'm confused: if you aren't running any fortran jobs, then why is there information in the yaml file that seems to be needed for compiling fortran?

Reply:
We are compiling fortran jobs, but I encountered problems when I attempted to run them.  Compiling fortran allows us to include several tests that compile but do not run fortran codes.

Reply:
Okay, thanks. I'm still not totally clear on this, though. Does this option imply that the system tests (class Z, if I remember correctly) are all built but not run, or does this also skip those system tests?

Reply:
(If possible, can you please answer that last question in the help description for this argument?)

Reply:
I have updated the --no-fortran-run description.   @billsacks Perhaps for your own edification you should review the 
tests that are run by selecting the "Checks" option at the top of this PR.

Reply:
Thanks, @jedwards4b . I don't see an updated description; maybe you haven't pushed it back yet? I do see, based on the checks and a second look at the code, that `--no-fortran-run` implies `--fast`, so simply noting that in the description would be sufficient (if you haven't already done so).

Reply:
oops - pushed now.